products/compute/terraform.yaml

```diff
@@ -2834,6 +2834,14 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       post_create: templates/terraform/post_create/labels.erb
   Zone: !ruby/object:Overrides::Terraform::ResourceOverride
     exclude: true
+  TargetGrpcProxy: !ruby/object:Overrides::Terraform::ResourceOverride
+    properties:
+      urlMap: !ruby/object:Overrides::Terraform::PropertyOverride
+        diff_suppress_func: 'compareSelfLinkOrResourceName'
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "target_grpc_proxy_basic"
```

Comment:
In addition to this, we'll want to write a handwritten update test for the resource.